firedrake/utility_meshes.py

```diff
@@ -716,6 +716,9 @@ def BoxMesh(nx, ny, nz, Lx, Ly, Lz, reorder=None, distribution_parameters=None,
     :arg Lx: The extent in the x direction
     :arg Ly: The extent in the y direction
     :arg Lz: The extent in the z direction
+    :arg diagonal: Two ways of cutting hexadra, should be cut into 6
+        tetrahedra (``"default"``), or 5 tetrahedra thus less biased
+        (``"crossed"``)
```

Comment:
Perhaps add a comment with a link to pictures?

Reply:
One can test using the following code:

from firedrake import *
#mesh = BoxMesh(1,1,1,1,1,1,diagonal="default")
mesh = BoxMesh(1,1,1,1,1,1,diagonal="crossed")
v = Function(FunctionSpace(mesh, "CG", 1))
File("/tmp/check_mesh.pvd").write(v)

Using the default cutting, one can see a mesh bias giving an ellipse like this

![ellipse](https://user-images.githubusercontent.com/48932582/87138664-ab1e6400-c296-11ea-8970-b41b8ece167c.png)

If we change to use "crossed" cutting, we can get a less-biased circle like this

![circle](https://user-images.githubusercontent.com/48932582/87138737-c8533280-c296-11ea-8618-958ea135b618.png)

Reply:
Just to be clear: these are pictures of simulations on the two grids that are not reproduced by Jingmin's snippet (of course).